nova_vm/src/ecmascript/execution/default_host_hooks.rs

```diff
@@ -29,4 +29,8 @@ impl HostHooks for DefaultHostHooks {
     fn enqueue_promise_job(&self, _job: Job) {
         // No-op
     }
+
+    fn get_storage_handle(&self) -> std::cell::RefMut<anymap::AnyMap> {
```

Comment:
**nitpick:** Is this actually a "handle"? Wouldn't that depend on the host? Also I'd suggest making this just an `Any` instead of `AnyMap`.

Likely the embedder knows exactly what sort of stuff they're storing, so they'll likely have just some `struct MyHostStorage` for efficient access of sub-parts.

**suggestion:** I guess maybe it might be a good option to make this return an `Option` and then make the default implementation just always return `None`.

Reply:
> suggestion: I guess maybe it might be a good option to make this return an Option and then make the default implementation just always return None.

I actually thought about doing this, but I thought it would just make every function that  reads the handler unnecessarily more verbose for little benefit

Reply:
> Wouldn't that depend on the host

I might as well just call it `get_host_data` to avoid assuming it is just for storage then, wdyt?

Reply:
Yeah, that's true. Not sure which one I prefer, really.

Reply:
> I might as well just call it `get_host_data` to avoid assuming it is just for storage then, wdyt?

Yeah. Please add a doc comment (`///`) in the method that it panics if the host has not overridden the default hook.